openxr/examples/vulkan.rs

```diff
@@ -47,11 +52,19 @@ fn main() {
     // If a required extension isn't present, you want to ditch out here! It's possible something
     // like your rendering API might not be provided by the active runtime. APIs like OpenGL don't
     // have universal support.
-    assert!(available_extensions.khr_vulkan_enable2);
+    assert!(available_extensions.khr_vulkan_enable || available_extensions.khr_vulkan_enable2);
 
     // Initialize OpenXR with the extensions we've found!
     let mut enabled_extensions = xr::ExtensionSet::default();
-    enabled_extensions.khr_vulkan_enable2 = true;
+    if available_extensions.khr_vulkan_enable2 {
+        enabled_extensions.khr_vulkan_enable2 = true;
+    } else {
+        enabled_extensions.khr_vulkan_enable = true;
+    }
+    #[cfg(target_os = "android")]
+    {
+        enabled_extensions.khr_android_create_instance = true;
+    }
```

Comment:
Style nit, since more work is needed anyway:
```suggestion
    enabled_extensions.khr_android_create_instance = cfg!(target_os = "android");
```

Reply:
This doesn't work because `khr_android_create_instance` field itself is cfg-gated to Android only.

Reply:
Oh, right.